web/templates/finance/bank_account_activities_return.html

```diff
@@ -0,0 +1,46 @@
+{#
+SPDX-FileCopyrightText: 2024 Gregor Düster <git@gdstr.eu>
+
+SPDX-License-Identifier: Apache-2.0
+#}
```

Comment:
Can you remove this header and/or replcace it with the header we usually use?
I am open to using more formal headers but that is a separate discussion.
(Also perhaps enforcing this via pre-commit hook might be a good idea)